### DISCUSSION TITLE
date differences formula

### DISCUSSION THREAD

**Discussion by anmola035** [mito-ds/mito#12] [Feature Requests]
Hey colleagues; 

I have two ideas:

 1. Add other currencies 
 2. Add the function of differences between dates.
  
  This is very useful in business. 

Thanks a lot!

**Comment by aarondr77** [comment]
Thanks for sharing these ideas! @anmola035. I'm going to break them into two separate discussion topics to make tracking them easier.

**Reply by aarondr77** [reply]
Moved point 1 (Add other currencies) to new discussion topic, here: https://github.com/mito-ds/monorepo/discussions/14

**Comment by aarondr77** [comment]
@anmola035 There are two different types of date differences. 1) Taking the difference between dates in the different columns, but  the same row 2) Taking the difference between dates that are in different columns & different rows.

Are you referring to a specific option in this feature request?

Currently, Mito does support option 1: Taking the difference between dates in the different columns, but  the same row. Although there isn't a spreadsheet formula for this operation, you can just use the '-' operator (see the example below). It could very well be the case that using the minus sign operator wasn't intuitive and we should add a formula to make it more discoverable :) 

<img width="394" alt="Screen Shot 2022-01-22 at 5 15 43 PM" src="https://user-images.githubusercontent.com/18709905/150657170-fd64e18d-fb54-4779-a9ac-50ca6c6b53e7.png">

Mito does not support option 2: Taking the difference between dates that are in different columns & different rows. The ability to do this relies on rolling window  / offset calculations, which Mito doesn't currently support.

**Reply by anmola035** [reply]
Hi Aaron,
Yes, I refer to the option 2, because the dates can be in differences tables, such as invoice date and pay date.
Thanks!!!
